test/actions/addTokensToVault.test.ts

```diff
@@ -0,0 +1,90 @@
+import BN from 'bn.js';
+import { Transaction } from '@metaplex-foundation/mpl-core';
+import { AccountLayout, MintLayout } from '@solana/spl-token';
+import { Keypair, sendAndConfirmTransaction } from '@solana/web3.js';
+
+import { Connection, NodeWallet } from '../../src';
+import { FEE_PAYER, pause, NETWORK } from '../utils';
+import { CreateMint, CreateTokenAccount, MintTo } from '../../src/programs';
+import { addTokensToVault, createExternalPriceAccount, createVault } from '../../src/actions';
+
+describe('addTokensToVault action', () => {
+  const connection = new Connection(NETWORK);
```

Comment:
please use localnet for tests
https://github.com/metaplex-foundation/js/blob/main/test/actions/createExternalPriceAccountLocal.test.ts

Reply:
upd